temperature/router.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from dotenv import load_dotenv
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from dependencies import get_db
+from . import crud, schemas
+from .weather import get_weather_temperatures
+
+load_dotenv()
+
+router = APIRouter()
+
+WEATHER_API_KEY = os.environ["WEATHER_API_KEY"]
+URL = "https://api.weatherapi.com/v1/current.json"
```

Comment:
could u get it from .env file?
also why do u need this consts here ?

Reply:
It`s like base url for API, then i can add any params to url